projects/plugins/jetpack/modules/masterbar/admin-color-schemes/colors/midnight/colors.scss

```diff
@@ -17,6 +17,13 @@ $color-primary-light: #007fde; // Calypso --color-primary-light
 $color-surface: $studio-white; // Calypso --color-surface
 $color-neutral-100-rgb: $studio-gray-100-rgb; // Calypso --color-neutral-100-rgb
 
+// Sidebar Nudges
+$menu-nudge-background: $studio-white;
+$menu-nudge-text-color: $studio-black;
+$menu-nudge-cta-background: $highlight-color;
+$menu-nudge-cta-color: $studio-white;
+$menu-nudge-cta-background-hover: #2a4ce7;
```

Comment:
This is not the same color as Calypso.

Calypso | WP Admin
--- | ---
<img width="274" alt="Screen Shot 2021-05-13 at 15 50 02" src="https://user-images.githubusercontent.com/1233880/118135132-f3506a80-b402-11eb-9166-f0404475f647.png"> | <img width="273" alt="Screen Shot 2021-05-13 at 15 49 54" src="https://user-images.githubusercontent.com/1233880/118135155-f8adb500-b402-11eb-8045-cea26ce76b86.png">

```suggestion
$menu-nudge-cta-background-hover: #00417d;
```

Reply:
🌈! e8b68bc